src/MicrosoftTeams.ts

```diff
@@ -54,6 +54,7 @@ namespace microsoftTeams {
     }
 
     export interface TabInstance {
+        appId: string;
```

Comment:
Do we need to add this or should we just return the appid as part of the result but not add it to the official TabInstance  interface?